src/main/resources/org/datadog/jenkins/plugins/datadog/configuration/DatadogAgentConfiguration/config.jelly

```diff
@@ -25,4 +25,8 @@
         method="checkTraceConnectivity" with="agentHost,agentTraceCollectionPort"
     />
 
+    <f:entry title="Datadog Site" field="site">
+      <f:select/>
+    </f:entry>
+
```

Comment:
This new entry is under `DatadogAgentConfiguration` namespace. Is there any way to send data without using the agent?

Reply:
We only need the user to specify the site manually if they're reporting through the agent. For the agentless mode we can deduce the site automatically, which is why I've only added this setting to the agent section.

Reply:
This is not relevant anymore, now that I've added the dedicated field for Datadog App hostname (see the conversation above), so I reverted this addition.